helm/fineract/templates/fineract-secret.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+kind: Secret
+apiVersion: v1
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
Use `{{ .Release.Name }}-secret`.

Reply:
Actually, it's already Kind = Secret so I think that might be a bit redundant...

Reply:
Yuh, but you have used the prefix `{{ .Release.Name }}` for other resources. So instead of having just the `{{ .Release.Name }}` I would suggest to use `{{ .Release.Name }}-secret` as the secret name. That'll be something meaningful. But it's not a requirement with high priority. Have you done a release for this implementation? Hope this works fine :) The configurations you've done look good to me